rascal-lsp/src/main/rascal/library/demo/lang/pico/LanguageServer.rsc

```diff
@@ -38,9 +38,17 @@ import util::IDEServices;
 import ParseTree;
 import util::ParseErrorRecovery;
 import util::Reflective;
-import lang::pico::\syntax::Main;
+extend lang::pico::\syntax::Main;
 import DateTime;
 
+syntax IdType
```

Comment:
add a small comment that for the purpose of demoing call-hierarchies we're extending the pico language that's defined in the standard library. Just to warn people that normally you wouldn't be adding language extensions in this file.